azure_arc_data_jumpstart/microk8s/azure/arm_template/ubuntuMicrok8s.json

```
@@ -0,0 +1,260 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "myIpAddress": {
+      "type": "string",
+      "metadata": {
+        "description": "Your public IP address, used to RDP to the client VM"
+      }
+    },
+    "vmName": {
+      "type": "string",
+      "defaultValue": "Arc-Data-Microk8s-K8s",
+      "metadata": {
+        "description": "The name of you Virtual Machine."
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "defaultValue": "arcdemo",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "sshRSAPublicKey": {
+      "type": "securestring",
+      "metadata": {
+        "description": "SSH Key for the Virtual Machine. SSH key is recommended over password."
+      }
+    },
+    "ubuntuOSVersion": {
+      "type": "string",
+      "defaultValue": "18.04-LTS",
+      "allowedValues": ["18.04-LTS"],
+      "metadata": {
+        "description": "The Ubuntu version for the VM. This will pick a fully patched image of this given Ubuntu version."
+      }
+    },
+    "azureLocation": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_D8s_v3",
+      "metadata": {
+        "description": "The size of the VM"
+      }
+    },
+    "virtualNetworkName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the VNet"
+      }
+    },
+    "subnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the subnet in the virtual network"
+      }
+    },
+    "networkSecurityGroupName": {
+      "type": "string",
+      "defaultValue": "Arc-Data-Microk8s-K8s-NSG",
+      "metadata": {
+        "description": "Name of the Network Security Group"
+      }
+    },
+    "resourceTags": {
+      "type": "object",
+      "defaultValue": {
+        "Project": "jumpstart_azure_arc_data"
```

```suggestion
        "Project": "jumpstart_azure_arc_data_services"
```

Make sure this is the tag across the board wherever tags are declared